tests/integration-tests/clusters_factory.py

```diff
@@ -86,6 +86,9 @@ def update(self, config_file, raise_on_error=True, log_error=True, **kwargs):
                 command.extend([f"--{kebab_case(k)}"] + list(map(str, val)))
             else:
                 command.extend([f"--{kebab_case(k)}", str(val)])
+        # TODO Remove the below validator suppression once the plugin scheduler will be officially supported
```

Comment:
Write:
`# TODO Remove the validator suppression below once the plugin scheduler is officially supported`